modules/notification/base/notifier.go

```diff
@@ -46,6 +46,9 @@ type Notifier interface {
 		issue *issues_model.Issue, comment *issues_model.Comment, mentions []*user_model.User)
 	NotifyUpdateComment(*user_model.User, *issues_model.Comment, string)
 	NotifyDeleteComment(*user_model.User, *issues_model.Comment)
+	NotifyNewWikiPage(doer *user_model.User, repo *repo_model.Repository, page, comment string)
+	NotifyEditWikiPage(doer *user_model.User, repo *repo_model.Repository, page, comment string)
```

Comment:
Update is a better word than Edit here imo

Reply:
I also thought about that when naming it. There is an "```UpdateRelease```" and an "```UpdateComment```", so It would be consistent with that. But this is the only place where I find the word "Update", there are other "synonyms" in use like "Change", which makes semantically more sense in that cases. So I don't "break the rule" in naming the notifier for editing a wiki page ```EditWikiPage``` to keep it semantically consistent. Take a look at all the notifiers ([here](https://github.com/go-gitea/gitea/blob/ed4774588086f4f493f717e5bb3f4950e1f525c3/modules/notification/base/notifier.go#L19-L63)). The names follow semantic names instead of consistent "CRUD"-like names. If ```EditWikiPage``` should be renamed to ```UpdateWikiPage```, I also need to rename ```NewWikiPage``` to ```CreateWikiPage``` to bring it in line with _some_ other actions. What do you think @lunny?